tests/test_data_processing.py

```diff
@@ -26,14 +38,512 @@
 )
 
 
+def test_get_optional_required_header():
+    """
+    This test checks whether header of
+    1. scalars and
+    2. time series
+    are returned correctly
+
+    In a third test it is checked whether a ValueError
+    is raised if an invalid string is passed with
+    data_type
+
+    """
+    # 1. Test
```

Comment:
Please do not test 3 things in one test function, but rather one thing  in one test. If there are variables you would like to share among test functions, you can create test classes with the methods testing one thing at a time.

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/57/commits/10429062c7180571172a6d83d4628f3da593b5ba.